actions/class.ResultServerStateFull.php

```diff
@@ -35,23 +37,20 @@ class taoResultServer_actions_ResultServerStateFull extends tao_actions_SaSModul
 
     /**
      * constructor: initialize the service and the default data
-     * @return Delivery
+     * @security("hide");
      */
     public function __construct() {
         parent::__construct();
         $this->service = $this->getClassService();
     }
 
-    public function getClassService() {
-        return taoResultServer_models_classes_ResultServerStateFull::singleton();
-    }
-
     /**
-     * @see TaoModule::getRootClass
-     * @return core_kernel_classes_Classes
+     * @return tao_models_classes_ClassService
+     * @security("hide");
      */
-    protected function getRootClass() {
-        
+    public function getClassService() {
+        // todo It seems that this method could be removed
```

Comment:
Please add an `@` before todo. That way PHP storm can pick it up in a todo overview:
`// @todo It seems that this method could be removed`